app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
When the document page is long, the menu is in the middle of the page, see

![image](https://user-images.githubusercontent.com/8007722/71860868-59680780-30c2-11ea-98f8-e790acd46f1e.png)